zio/zio.go

```diff
@@ -1,16 +1,28 @@
 package zio
 
 import (
+	"flag"
 	"io"
 
 	"github.com/brimsec/zq/zbuf"
 )
 
+// This is clumsy, it has the union of the flags accepted by all
+// the different Writer classes.
 type Flags struct {
 	UTF8       bool
 	ShowTypes  bool
 	ShowFields bool
 	EpochDates bool
+	FrameSize  int
+}
+
+func (f *Flags) SetFlags(fs *flag.FlagSet) {
+	fs.BoolVar(&f.ShowTypes, "T", false, "display field types in text output")
+	fs.BoolVar(&f.ShowFields, "F", false, "display field names in text output")
+	fs.BoolVar(&f.EpochDates, "E", false, "display epoch timestamps in text output")
+	fs.BoolVar(&f.UTF8, "U", false, "display zeek strings as UTF-8")
+	fs.IntVar(&f.FrameSize, "b", 0, "BZNG frame size (0 for no frames)")
```

Comment:
Usage message should identify the units. Also, they're records now, but would bytes be more useful?

Reply:
> would bytes be more useful?

I don't know, using bytes would be complicated by compression if/when it is added.  It appears to me that this is simpler and meets our immediate needs for indexing...

Reply:
👍